examples/stickysession/nginx/README.md

```diff
@@ -0,0 +1,58 @@
+# Sticky Session
+
+This example demonstrates how to Stickness in a Ingress.
```

Comment:
Suggest: how to achieve session affinity using cookies
Though unlike haproxy we're not allowed to pick the cookie right? nginx just uses a hash of the request IIRC